test/SILGen/package_allow_non_resilient_access.swift

```diff
@@ -44,7 +44,7 @@
 // RUN:   -experimental-allow-non-resilient-access \
 // RUN:   -emit-module -emit-module-path %t/Utils.swiftmodule \
 // RUN: 2>&1 | %FileCheck %s --check-prefix=CHECK-DIAG-1
-// CHECK-DIAG-1: warning: ignoring -experimental-skip-non-exportable-decls (overriden by -allow-non-resilient-access)
+// CHECK-DIAG-1: warning: ignoring -experimental-skip-non-exportable-decls (overridden by -allow-non-resilient-access)
```

Comment:
This is a FileCheck directive that checks for an occurrence of the given string, not just a comment, so you need to update the warning message too (`warn_ignore_option_overriden_by`). 


Ditto other changes in this file.

Reply:
I reverted changes related to that in this file. I did not quite understand what you suggested I change, and I saw references to `warn_ignore_option_overriden_by` in some other files, so felt best revert:

<img width="616" alt="Screenshot 2024-08-08 at 19 25 39" src="https://github.com/user-attachments/assets/a738270a-5e76-4dc6-a360-7f0a6939bc11">